config/schemas/config.yaml

```
@@ -4974,6 +4974,12 @@ properties:
             type: object
             additionalProperties: false
             properties:
+              azcopyBufferGB:
+                type: number
+                description: Memory buffer size in GB for azcopy CLI
```

```suggestion
                description: Configure the memory buffer size in GB for Azure copy operations.
```
And similarly for the concurrency configuration.